Observer/CheckoutCartIndex.php

```diff
@@ -57,31 +57,31 @@ class CheckoutCartIndex implements ObserverInterface
     /**
      * Constructor
      *
-     * @param \Magento\Framework\App\Action\Context $context
-     * @param \Magento\Checkout\Model\Session $session
+     * @param \Magento\Framework\App\Action\Context       $context
+     * @param \Magento\Checkout\Model\Session             $session
      * @param \Magento\Framework\Message\ManagerInterface $messageManager
-     * @param \Magento\Quote\Model\QuoteFactory $quoteFactory
-     * @param \Magento\Sales\Model\OrderFactory $orderFactory
-     * @param Coupon $coupon
-     * @param CouponUsage $couponUsage
-     * @param StockManagementInterface $stockManagement
-     * @param SystemConfig $systemConfig
+     * @param \Magento\Quote\Model\QuoteFactory           $quoteFactory
```

Comment:
import lasses (use) so we dont have the FQN here

Reply:
Done